example/tree_shake.dart

```diff
@@ -1,42 +0,0 @@
-// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I use these files in `tool/check_tree_shaking.dart`, so they should be moved below `tool/src/` somewhere, probably `tool/src/tree_shaking/*.dart` (and update the `check_tree_shaking.dart` file to point there).

I can do that if you don't delete the files.